lightning/restore/restore.go

```diff
@@ -1266,13 +1277,20 @@ func newChunkRestore(
 		return nil, errors.Trace(err)
 	}
 
+	routeRes := router.Route(chunk.Key.Path)
+	if routeRes == nil || routeRes.Type == mydump.SourceTypeIgnore {
+		return nil, errors.Errorf("file '%s' is filtered by file router", chunk.Key.Path)
+	}
```

Comment:
wait i thought the point of saving compression/type/key into the chunk checkpoint is the avoid routing again? 🤔

Reply:
oo, I forgot to update these logic